apps/gdalinfo_lib.cpp

```diff
@@ -440,7 +463,10 @@ char *GDALInfo( GDALDatasetH hDataset, const GDALInfoOptions *psOptions )
             if( bJson )
             {
                 json_object *proj4 = json_object_new_string(pszProj4);
+                json_object *poStacProj4 = nullptr;
+                json_object_deep_copy(proj4, &poStacProj4, nullptr);
                 json_object_object_add(poCoordinateSystem, "proj4", proj4);
+                json_object_object_add(poStac, "proj:projjson", poStacProj4);
```

Comment:
`proj:projjson` should be https://proj.org/specifications/projjson.html not proj4 strings

Reply:
Yup will fix, thanks...